src/ngrest/base/ActiveWindow.php

```diff
@@ -375,6 +376,28 @@ public function getCondition()
         return empty($this->_condition) ? '' : $this->_condition;
     }
     
+    private $_permissionLevel = null;

```

Comment:
private $_permissionLevel;

is equals to

private $_permissionLevel = null;

So no need for default value definition.

Reply:
Ok, wanted to be sure to default the value to null.. wasn't sure about the default value if no thing is specified. So will update accordingly.

Reply:
@nadar this actually makes me wondering if it wouldn't be good to set the default permission level to Auth::CAN_UPDATE this way we will not break backward compatibility.. What do you suggest ?

Reply:
Thanks for pointing into that direction, yes indeed we should set can_update in order to prevent BC! Thanks